### ISSUE TITLE
add signup page for paid service

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#619]
Add a page where people can register their card details for paid features.

* [x] write copy
* [x] design form
* [x] connect stripe
* [x] layout page

**Comment by equivalentideas** [comment]
Content first pass:

![screen shot 2015-02-04 at 11 36 02 am](https://cloud.githubusercontent.com/assets/1239550/6032839/3127e59a-ac62-11e4-8b90-7c32f6bcf727.png)

Could I please get some quick reactions to this copy/content @mlandauer @henare @kat ?

**Comment by mlandauer** [comment]
@equivalentideas looks good! As an alternative to explore is there another way to describe it in terms of the benefits of the powerful new tools?

**Comment by equivalentideas** [comment]
Nice @mlandauer . I think in:

> Subscribe now for powerful new tools to **manage you alerts**

“manage your alerts” isn't a real goal, it's a stepping stone, and we could mention the benefits of doing this instead. The reason to 'manage your alerts' is to better focus them: reduce noise, increase signal. By focusing the value and scope of the information they get they'll save time, be more on top of the info they want and get more value.

> Subscribe now for powerful new tools to focus your alerts.

This relates to managing all your alerts through one interface for starters, but also to new ways to scope requests etc. .

**Comment by henare** [comment]
@equivalentideas looks good :bear: 

>Subscribe now for powerful new tools to focus your alerts.

Still sounds like a stepping stone and doesn't describe the benefits to me.

**Comment by equivalentideas** [comment]
> Still sounds like a stepping stone and doesn't describe the benefits to me.

@henare good point.

The real benefits to getting more focused, timely information are a competitive advantage.

What about doubling down on that benefit in the second sentence and also hint at what the tools are:

> The property professionals who use PlanningAlerts are the first to know about new DAs and subdivisions they care about.
> **Stay ahead with powerful new tools to focus your alerts.**

![screen shot 2015-02-04 at 12 37 07 pm](https://cloud.githubusercontent.com/assets/1239550/6033496/90b1f692-ac6a-11e4-8395-e75e9f4c7c18.png)

**Comment by henare** [comment]
I think that's an improvement.

"focus your alerts" feels a bit limp to me. Can we say something broader and more exciting?

**Comment by equivalentideas** [comment]
What about dropping it to “Stay ahead with powerful new tools”?

**Comment by equivalentideas** [comment]
@mlandauer can I get your thoughts on that too?

**Comment by mlandauer** [comment]
You're describing why people use the free planningalerts and what the benefits are. What about the extra benefits of the paid service? Right now the distinction is a bit blurry.

**Comment by equivalentideas** [comment]
That's just what I needed @mlandauer :)

The difference between people with the new features and those without is that the people with them can more easily control what information they receive, which will allow them to get better information, a greater competitive advantage.

Two ideas on that:

> Property professionals who use PlanningAlerts are the first to know about new DAs and subdivisions they care about.
> Stay ahead with powerful new tools to maximise the return from your alerts.

> Property professionals who use PlanningAlerts are the first to know about new DAs and subdivisions they care about.
> Stay ahead with powerful new tools to get better, more focused results.

**Comment by equivalentideas** [comment]
Here's a first pass of the signup page content:

![screen shot 2015-02-04 at 4 33 28 pm](https://cloud.githubusercontent.com/assets/1239550/6035669/aa0990f2-ac8b-11e4-9c38-0e8fd62f3991.png)

Anyone have feedback?

I'm going to move on to connecting Stripe.

**Comment by mlandauer** [comment]
I think the credit card details should be in a modal and "submit payment"
doesn't seem right

**Comment by equivalentideas** [comment]
Why do you think the modal?

I think 'submit payment' is very clear (here's an example https://upcase.com/professional/checkouts/new ), but what did you have in mind?

**Comment by mlandauer** [comment]
Because that form is completely distracting from the message. Look at what stripe does with its pre-baked javascript modal form which has lots of ui niceties.

**Comment by mlandauer** [comment]
https://stripe.com/docs/checkout

**Comment by equivalentideas** [comment]
That's quite nice, I'll give it a go.

**Comment by equivalentideas** [comment]
Yep that's working nicely. Works like the [demo](https://stripe.com/checkout).

I've done the basic setup of it, now I need to connect it to stripe. Configuration/style iterations should happen once it's working.

I think it's time for #622 @mlandauer @henare

**Comment by kat** [comment]
First I thought 'tune planning alerts to suit your business' then I thought that was really dull. 

I'm not feeling any excitement or immediacy in the tone of these ideas. Nothing make me want to click the button.

So looking at it another way, let's have some fun and loosen this discussion up a bit - it's February right? Nearly valentine's day! :love_letter:  

We know you love PlanningAlerts
:cupid:
You'll love powerful new tools a whole lot more.
Sign up now! Early access, only $99

**Comment by kat** [comment]
For the button, instead of 'submit', how about "Pay Now" or "Sign Up" or "Create Account" 
or even, take the $amount out of the earlier section and then use "Pay $99/month" for the big action button.
also should there be something about not being locked in for a timeframe.

**Comment by kat** [comment]
ok here's a revision. 

1000's of Property Professionals love PlanningAlerts to find out first about subdivisions they care about.
This is your invitation to access powerful new tools. 
Make the most of your alerts.
Sign up now! Early access,
Only $99/month

**Comment by equivalentideas** [comment]
Thanks for that @kat just coming back around to the text now so that's awesome.

**Comment by kat** [comment]
cool!  So it's totally based on what you had already, I just shook it up a bit. 
The revision attempts to exude some love/energy, a clear proposition, speak directly to the sender, and in plain English. Not quite there yet ... @equivalentideas let's see some leapfrogging :-)

**Comment by equivalentideas** [comment]
So with the stripe plugin form I've made the price/subscribe thing the button. Currently still all grey and boring—but works!

![screen shot 2015-02-05 at 3 42 52 pm](https://cloud.githubusercontent.com/assets/1239550/6055169/fa5dd884-ad4d-11e4-87a2-b9d84a743bda.png)

Here's where I'm up to with this:

* Stripe integration is working ace and will just need the keys entered when ready.
* Coming back now to the text, for both the signup and success pages.
* Then adding colour/polishing.

## Success page

What do we need to say here? Thoughts on this MVP @mlandauer @henare @kat ?

Things we could say:

* You've successfully subscribed, with x email address.
* We the new features are in development and you'll be the first to know
* We won't charge you anything until you start using the service.
* 'Have colleagues who'd want first access to the new tools?' Email them invite..

**Comment by equivalentideas** [comment]
## Copy

This is specifically for Real Estate Agents using PlanningAlerts is multiple subscriptions. They are between 30 and 70, most likely in there 50s. They write unemotionally and specifically about the value of the service, in contrast to citizens who don't use the site in there work.

@kat I think we can make it more direct, like your draft above, particularly with the title.

For this experiment I think we focus on the value to them and their desire to be competitive against other agents, rather than the emotional connection to the app which I don't think we have a lot of evidence for. 

A clear, specific tone matches the way they write about PA, I think we add confidence to make it clear they should join it for their benefit, not ours.

**Comment by equivalentideas** [comment]
Success page draft copy for thoughts please:

# Success!

The subscription for luke@equivalentideas.com has been confirmed. Thank you.

**The new alert management tools will be available soon and you will be first to get access.** 
You will not be charged before then.

If you have any questions or feedback on PlanningAlerts please don’t hesitate to contact@planningalerts.org.au.

**Comment by equivalentideas** [comment]
I'm working on styling the button and layout of the success page while waiting on feedback on content of the success page.

I'm happy with the current sign up page copy for this audience. Please note the **Copy** bit above to see thinking on that, more ideas very welcome. There's also the email copy to think about :)

**Comment by kat** [comment]
@equivalentideas  Yes, its' getting there. Good. Though I think you misunderstand the intent with the tone. Yes it's professional. Identifying value and encouraging competitive streak are good observations, Is there a way to key the emotion into this appropriately?  Getting the emotion right is important in closing the sale. *click!*

Maybe you're using it because its a word which resonates in this sector, but I'm not sure how they would be 'maximising' their alerts? Can you explain what that means please?

$99/month is much better at the end, but doesn't work on the button, sorry.

**Comment by equivalentideas** [comment]
> Maybe you're using it because its a word which resonates in this sector, but I'm not sure how they would be 'maximising' their alerts? Can you explain what that means please?

"Maximise the return from your alerts" is saying 'get the most from your alerts' in the language of property sales. Manage alerts more effectively to reduce noise and increase signal—making them more effective. :signal_strength: 

> $99/month is much better at the end, but doesn't work on the button, sorry.

Cool, I've busted it out. (with this feedback, could you include the 'why' :stuck_out_tongue_winking_eye:  it's slower to work through without it).
![screen shot 2015-02-05 at 6 07 22 pm](https://cloud.githubusercontent.com/assets/1239550/6056244/e3cb1c76-ad61-11e4-9ab2-081cbda39dae.png)

> Is there a way to key the emotion into this appropriately?

I think the current emotional focus on their work will be effective, but I'm finding it hard to keep iterating on that :) . I'll include it in the PR and we can keep changing it.

**Comment by equivalentideas** [comment]
Latest version:
![screen shot 2015-02-05 at 6 35 51 pm](https://cloud.githubusercontent.com/assets/1239550/6056505/dfd5eff2-ad65-11e4-8c56-feaef4111dcc.png)

**Comment by kat** [comment]
Hi @equivalentideas Thanks for asking me to review this again. I've looked again at the first version and the latest. I'm glad the word The was removed at the beginning. Beyond that I don't feel like the feedback I gave has had any positive impact or helped improve this copy. If anything I feel have conspired to help make this text marginally worse rather than help vastly improve it. 

Given that it's now Friday afternoon, and we're not aiming to send today, I think this isn't a good time to try and pull apart what's not working in detail. I suggest we revisit in person on Monday with fresh eyes.

**Comment by equivalentideas** [comment]
> Given that it's now Friday afternoon, and we're not aiming to send today, I think this isn't a good time to try and pull apart what's not working in detail. I suggest we revisit in person on Monday with fresh eyes.

Sounds good :)

**Comment by equivalentideas** [comment]
Also the latest draft of the text is at https://github.com/openaustralia/planningalerts-app/pull/630#issuecomment-73182094 on #630 :

> Get more from your alerts

> Property professionals who use PlanningAlerts are the first to know about new DAs and subdivisions. Many of you have dozens of alerts set up to get greater insight in the areas you care about.

> We’re now adding powerful new tools so you can maximise the return from your alerts.

> Subscribe now for only $99/month.

**Comment by kat** [comment]
ok we've workshopped this, here's where we're at.

Tailor your alerts to suit your business needs

Access powerful new features from PlanningAlerts.
Track your key investments, keep important clients in the know and discover new opportunities. 

Subscribe now, $99/month

**Comment by kat** [comment]
shall we take this one for a spin?

**Comment by equivalentideas** [comment]
Trying leading with 'opportunities'.

> ## Tailor your alerts to suit your business needs

> Access powerful new features from PlanningAlerts.
> Discover new opportunities, track key investments and keep your important clients in the know. 

> Subscribe now, $99/month

**Comment by mlandauer** [comment]
@equivalentideas @kat I'm happy to go with that wording as a first iteration

**Comment by equivalentideas** [comment]
This was done ages ago.
